lib/project/ext/ui_view.rb

```diff
@@ -82,7 +82,12 @@ def image
   def stylesheet
     rmq.stylesheet
   end
+
   def stylesheet=(value)
     rmq.stylesheet = value
   end
+
+  def find(*args)
+    rmq(self).find(args)
```

Comment:
It probably works (because @twerth is a fantastic software engineer who [accounts for things like this](https://github.com/infinitered/rmq/blob/ca07631fe6845abf8da1aca6a0b555740928422c/motion/ruby_motion_query/selectors.rb#L60)), but the `*args` is more correct.